packages/ui-components/src/components/form/RadioGroup.stories.ts

```diff
@@ -0,0 +1,65 @@
+import type { Meta, StoryObj } from '@storybook/vue3'
```

Comment:
1. The `Actions` panel in storybook is empty even tho events are being fired when clicking on the options. Ensure you set this up correctly.
2. You should use the `args['update:modelValue'](val); ctx.updateArgs({ ...args, modelValue: val })` pattern to ensure that the updated property also shows up updated in the storybook controls panel
3. You should add stories for: Pre-configuring a specific option; Validating the input w/ vee-validate; Showing a required badge (see other inputs for common stories)